packages/sdk/src/modules/moduleFromEndpoints/utils/getHttpClient.ts

```diff
@@ -0,0 +1,60 @@
+import { Options, HTTPClientConfig } from "../types";
+
+export const getHTTPClient = (options: Options) => {
+  const getUrl = (path: string, config: HTTPClientConfig): string => {
+    const { apiUrl, ssrApiUrl } = options;
+    const { method = "POST", params = [] } = config;
+
+    // Determine the base URL based on the environment
+    const baseUrl =
+      typeof window === "undefined" ? ssrApiUrl || apiUrl : apiUrl;
+
+    // Ensure the base URL ends with a slash
+    const normalizedBaseUrl = baseUrl.endsWith("/") ? baseUrl : `${baseUrl}/`;
+    const url = `${normalizedBaseUrl}${path}`;
```

Comment:
I think it would be safest to already use `new URL` at this stage. Concatenating url paths can be tricky and prone to errors

Reply:
I cannot use new URL cause it can be set to api but this comment realized me that I have that few lines below and the approach won't work properly with GET requests. Now it's fixed :)